common/xfer/plugins.go

```diff
@@ -0,0 +1,216 @@
+package xfer
```

Comment:
At least I would call it `pluginspecs` to make the separtion with `probe/plugins` more evident.